exoctk/phase_constraint_overlap/__init__.py

```diff
@@ -0,0 +1 @@
+from . import phase_constraint_overlap
```

Comment:
Is this something needed in an __init__ file? I never knew that!

Reply:
I don't think it's necessary but I do it out of habit. @bourque or @hover2pi do you know if it is still necessary to do this?

Reply:
I'm not really sure.  I would say that if it works we should just leave it for now.